tcl/tson.tcl

```diff
@@ -119,3 +119,19 @@ proc qc::tson2xml { tson } {
 	}
     }
 }
+
+proc qc::tson_get {tson args} {
+    #| Returns the value at the specified path in the TSON.
+    #| Requires PostgreSQL 9.3 or later.
+
+    # Construct a PostgreSQL array literal from the path.
+    set path "\{[join $args ","]\}"
```

Comment:
Not at the moment. There is [qc::sql_list2array](https://github.com/qcode-software/qcode-tcl/blob/master/tcl/sql.tcl#L184) however it produces an array constructor not a literal. The right-hand argument for the JSON operator must be an array literal.